applauncher/src/test/java/de/interseroh/tmb/applauncher/client/ApplauncherWebAppTest.java

```diff
@@ -36,7 +38,8 @@ public void setUp() throws Exception {
 	}
 
 	@Test
+	@Ignore
 	public void testOnModuleLoad() throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1607)